### PR TITLE
Add hook to retrieve container logs after run

### PR DESCRIPTION
**Pull request by fhoering**
- to be able to upload them somewhere from within the container

### CODE REVIEW

File: cluster_pack/skein/skein_config_builder.py [hunk 1]
```diff
@@ -22,17 +21,25 @@ def build_with_func(
...
-    -------
-    SkeinConfig
+    :param func: the function to execute remotely
+    :param args: the function's arguments
+    :param package_path: the path on distributed storage where to find the application package
+                         (pex, conda zip)
+    :param additional_files: additional files to ship to the cluster
+    :param tmp_dir: a temp dir for local files
+    :param log_level: default remote log level
+    :param log_output_path: the path on distributed storage where to upload the container logs
```
Comment (t-henri) [inline cluster_pack/skein/skein_config_builder.py:38]: It doesn't match with the function definition, the last parameter in process_logs and it is not a path

File: cluster_pack/skein/skein_config_builder.py [hunk 2]
```diff
@@ -47,25 +54,31 @@ def build_with_func(
...
-    -------
-    SkeinConfig
-
+    :param module_name: the module to execute remotely
+    :param args: the module's cli arguments
+    :param package_path: the path on distributed storage where to find the application package
+                         (pex, conda zip)
+    :param additional_files: additional files to ship to the cluster
+    :param tmp_dir: a temp dir for local files
+    :param log_output_path: the path on distributed storage where to upload the container logs
```
Comment (t-henri) [inline cluster_pack/skein/skein_config_builder.py:80]: Same than for build_with_fund

File: cluster_pack/skein/skein_config_builder.py (hunk 1 above)
Comment (fhoering) [reply cluster_pack/skein/skein_config_builder.py]: done

File: cluster_pack/skein/skein_config_builder.py (hunk 2 above)
Comment (fhoering) [reply cluster_pack/skein/skein_config_builder.py]: done

**Pull request merged**